src/api/telemetry/TelemetryCollection.js

```diff
@@ -322,7 +322,7 @@ export class TelemetryCollection extends EventEmitter {
      * @private
      */
     _setTimeSystem(timeSystem) {
-        let domains = this.metadata.valuesForHints(['domain']);
+        let domains = this.metadata ? this.metadata.valuesForHints(['domain']) : [];
```

Comment:
Can we set domain to it's default of `[]` right off the bat? Then can we just get one check for this.metada and if it exists, then we can set domain to that (as well as metadatavalue as it also relies on this.metadata).

Something like:
``` js
let domains = [];
let metadataValue = { format: timeSystem.key };

if (this.metadata) { ... update if necessary }
```